### PR TITLE
Add SQS Sample (#508)

### LINKED ISSUE #508: Add SQS sample

### PR DESCRIPTION
**Pull request by tomazfernandes**
A simple example sending and receiving a message with SQS and CDK infrastructure.

Fixes #508.

### CODE REVIEW

**Comment by tomazfernandes** [comment]
We can add more samples later if that's the case - at least everything is set up. Let me know if you have any suggestions!

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsSample.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	private static final Logger LOGGER = LoggerFactory.getLogger(SpringSqsSample.class);
+
+	private static final String QUEUE_NAME = "test-queue";
+
+	public static void main(String[] args) {
+		SpringApplication.run(SpringSqsSample.class, args);
+	}
+
+	@SqsListener(queueNames = QUEUE_NAME)
+	void listen(String message) {
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsSample.java:41]: I think the sample could be a bit more sophisticated - passing a JSON as a payload, and reading message attributes in the listener method, just to showcase a little bit more than a bare minimum

Comment (tomazfernandes) [reply spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsSample.java]: Cool, how about now?

**Review by maciejwalkowiak — APPROVED** [review]
Perfecto!

**Pull request merged**